utils/bots/TutorBot/cogs/TutorStaff.py

```diff
@@ -224,6 +224,7 @@ async def schedule_t(
         guild_ids=[MAIN_ID.g_main, TUT_ID.g_tut]
     )
     @permissions.has_any_role(MAIN_ID.r_tutor, guild_id=MAIN_ID.g_main)
+    @permissions.has_any_role(860932484206952468, guild_id=TUT_ID.g_tut)
```

Comment:
Is it really worth it to hardcode here? Why not use the existing system and leave it to use ConfigCat?